python_files/generate_supplementary_plots.py

```diff
@@ -1,17 +1,119 @@
 # File with code for generating supplementary plots
+import math
 import os
+import pathlib
 import shutil
+from typing import Any, List, Optional, Tuple, Union
 
 import numpy as np
 import pandas as pd
 import matplotlib
+import natsort as ns
+import xarray as xr
+from PIL import Image, ImageDraw, ImageFont
+
+from alpineer.io_utils import list_folders, list_files, remove_file_extensions, validate_paths
+from alpineer.load_utils import load_imgs_from_tree
+from alpineer.misc_utils import verify_in_list
 
 matplotlib.rcParams['pdf.fonttype'] = 42
 matplotlib.rcParams['ps.fonttype'] = 42
 
 import matplotlib.pyplot as plt
 import seaborn as sns
 
+SUPPLEMENTARY_FIG_DIR = "/Volumes/Shared/Noah Greenwald/TONIC_Cohort/supplementary_figs"
+ACQUISITION_INDICES = [
+    11, 12, 13, 14, 15, 17, 18, 20, 22, 23, 24, 28, 29, 30, 31, 32, 33, 34, 35,
+    36, 39, 40, 41, 42, 43, 44, 45, 46, 47
+]
+
+
+def stitch_and_annotate_padded_img(image_data: xr.DataArray, padding: int = 25,
```

Comment:
Will you create a separate file to hold functions, and move both these to that file? Then import them here